### ISSUE TITLE
operator < viterbi_decode > error

### ISSUE THREAD

**Issue by fourseven471** [PaddlePaddle/Paddle#49437]
### 请提出你的问题 Please ask your question

静态图下viterbi_decode报错！
_, infers = paddle.text.viterbi_decode(logits, self.crf.transitions, text_a_lens, False)

<img width="863" alt="image" src="https://user-images.githubusercontent.com/48342616/209924063-625438fd-3811-4c6f-b648-03815ac60450.png">

**Comment by From00** [comment]
你好，看报错提示是输入tensor的shape信息有误。请检查下输入是否合法。
